src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Summary/_items.html.twig

```diff
@@ -16,8 +16,8 @@
             </tr>
             </thead>
             <tbody>
-                {% for item in cart.items %}
-                    {% include '@SyliusShop/Cart/Summary/_item.html.twig' with {'item': item, 'form': form.items[loop.index0]} %}
+                {% for index,item in cart.items %}
```

Comment:
I'm not sure about `index` keyword here, as it can be confused with `loop.index` in long-term. Perhaps `key` would be a better word? Ref: https://twig.symfony.com/doc/2.x/tags/for.html#iterating-over-keys-and-values

Reply:
👍 for using `key` and adding a space after comma would be great 🎉